dropshot/src/router.rs

```diff
@@ -146,11 +145,15 @@ impl PathSegment {
                 (var, None)
             };
 
+            /*
+             * Note that the only constraint on the variable name is that it is
+             * not empty. Consumers may choose odd names like '_' or 'type'
+             * that are not valid Rust identifiers and use rename them with
```

Comment:
```suggestion
             * that are not valid Rust identifiers and rename them with
```

Reply:
Thanks. I had something like "use serde(rename = ...)" and clearly only half-unwound that idea...